plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -871,7 +895,7 @@ private static ColumnMapping timeColumnMapping(int precision)
                 DISABLE_PUSHDOWN);
     }
 
-    public static LongWriteFunction timeWriteFunction(int precision)
+    private static LongWriteFunction timeWriteFunction(int precision)
```

Comment:
super duper nit: separate commit?

Reply:
I will revert this change in this pull request.

Reply:
https://github.com/trinodb/trino/pull/9946